ci-operator/step-registry/baremetal/lab/ipi/baremetal-lab-ipi-workflow.yaml

```diff
@@ -3,6 +3,7 @@ workflow:
   steps:
     pre:
       - chain: baremetal-lab-ipi-conf
+      - ref: baremetal-lab-ipi-conf-network-vips
```

Comment:
This step can be moved to the `baremetal-lab-ipi-conf` chain along with `baremetal-lab-upi-conf-network` step.